app/src/main/res/menu/menu_channel.xml

```diff
@@ -4,6 +4,12 @@
     xmlns:tools="http://schemas.android.com/tools"
     tools:context=".fragments.list.impl.ChannelFragment">
 
+    <item
+        android:id="@+id/menu_item_search"
```

Comment:
What's search doing in channel fragment?

Reply:
I hope someday this https://github.com/TeamNewPipe/NewPipeExtractor/issues/49 will be done and i can make a search in a selected channel